scripts/case_change_service.py

```diff
@@ -0,0 +1,42 @@
+# coding: utf-8
+#
+# Copyright 2020 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import absolute_import  # pylint: disable=import-only-modules
+from __future__ import unicode_literals  # pylint: disable=import-only-modules
+
+import re
+
+SNAKE_CASE_RE = re.compile(r'([^\-_\s])[\-_\s]+([^\-_\s])')
+
+def camelize_string(s):
```

Comment:
Please rename the variable so that it's clear what they stand for.

Reply:
Also, what happens if the string is already in camelCase?

Reply:
It returns the same string.